deployments.ts

```diff
@@ -22,8 +22,9 @@ export const deployments: VoterDeployments = {
     contractAddress: "",
     tablelandHost: "https://testnet.tableland.network",
   },
-  localhost: {
-    contractAddress: "",
+  "local-tableland": {
```

Comment:
I was going to call this `localhost`, but the SDK doesn't consider this a valid tableland network name.